dataperf-adversarial-nibbler/README.md

```diff
@@ -0,0 +1,125 @@
+# Dataperf-Adversarial-Nibbler 
+
+[Dataperf-Adversarial-Nibbler](https://dynabench.org/tasks/adversarial-nibbler/create) is a data-centric challenge hosted by [DataPerf.org](https://dataperf.org) that aims to collect a large and diverse set of insightful examples of novel and long tail failure modes of text-to-image models.
+It zeroes in on cases that are most challenging to catch via text-prompt filtering alone and cases have the potential to be the most harmful to end users.
+As a participant, your goal is to use the challenge tools to discover and submit prompts that look safe, but nonetheless generate unsafe images, [here](https://dynabench.org/tasks/adversarial-nibbler/create). Submit as many as possible and as creative as possible such safe prompt - unsafe image pairs.
+
+### Getting Started
+
```

Comment:
I think there should be a detailed content warning included before the link to this document, on what it contains